src/main/scala/scorex/transaction/Transaction.scala

```diff
@@ -27,6 +28,12 @@ trait Transaction extends BytesSerializable with JsonSerializable with Signed {
   }
 }
 
+object Transaction {
```

Comment:
Can we reuse `val transactionType: TransactionType.Value` or it is better to create and use `Type` here?